docs/development.md

```diff
@@ -31,6 +31,9 @@ npm i
 npm run build -ws
 ```
 
+**Note**: We have included a `package-lock.json` in `script/workspace`. If packages cannot be installed correctly re-run `script/boostrap` and run `npm ci` to
```

Comment:
Feel free to ignore. Just a thought but it might be too wordy.

```suggestion 
**Note**: We have included a `package-lock.json` in `script/workspace`. If `npm run build -ws` fails because packages are not installed correctly with `nmp i`, re-run `script/boostrap` and run `npm ci` to
```